packages/install-expo-modules/src/plugins/ios/__tests__/withSwiftVersion-test.ts

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs';
+import path from 'path';
+import xcode from 'xcode';
+
+import { setSwiftVersionIfNotPresent } from '../withSwiftVersion';
+
+const fixturesPath = path.resolve(__dirname, 'fixtures');
+
+it(
+  "given targets in pbxproj that don't have `SWIFT_VERSION` build config specified, " +
+    'setSwiftVersionIfNotPresent() sets it',
+  async () => {
+    const project = xcode.project(
+      path.join(fixturesPath, 'TestTargetsDontHaveSwiftVersion.pbxproj')
+    );
+    project.parseSync();
```

Comment:
I think we can use the `IOSConfig.XcodeUtils.getPbxproj` helper from `@expo/config-plugins` here, it should do the same without `install-expo-modules` having to depend on the `xcode` package.

Something like:

```
import {} from '@expo/config-plugins';

IOSConfig.XcodeUtils.getPbxproj(path.join(fixturePath, 'TestTargetsDontHaveSwitftVersion.pbxproj'))
```

But that might require the fixture to have an `ios` folder, e.g. `ios/TestTargetsDontHaveSwiftVersion.pbxproj`.

Reply:
ended up using `xcparse` which is the package that `withXCParseXcodeProjectBaseMod` is built on top of